CHANGELOG.md

```diff
@@ -1,3 +1,207 @@
+
+## Rectifying Clients Without Proper Chains
+
+Once an authenticator has been identified as problematic, an administrator should verify the client is using the newest
+possible versions of its SDK and either re-enroll it or request the identity to extend the next time it 
+authenticates. Both scenarios result in a new certificate and chain being provided to the client.
+
+- **Re-Enrollment** removes the current authenticator, making authentication impossible until enrollment is completed.
+  Once completed, a new authenticator is created, and the new certificate generated from the process can be used to 
+  authenticate. Humans or other external automation processes have to deliver and consume the new enrollment JWT.
+- **Extension** leaves the authenticator in place and authentication can still occur. When the extension process is
+  completed, the authenticator is updated and will use the new certificate generated from the process. SDKs will 
+  handle the process by raising SDK specific events to drive the process.
+
+While related and similar, using one over the other depends on the situation. Re-Enrollment is best for when
+an client or its host are unrecoverable; either through damage, decommissioning, or suspected to be compromised.
+Extension is useful when the client and host are in a known good status and one simply wants to issue new
+certificates to said client. Extension is also "hands off" from the client as long as the client is using 
+SDKs that support enrollment request/key rolling.
+
+## Useful Authenticator Values
+
+Authenticators are an entity within OpenZiti and can be queried via the CLI (`ziti edge list authenticators`) or
+Edge Management API (`GET https://<host>/edge/management/v1/authenticators`). Below are properties that are 
+useful for determining clients with improper chains.
+
+- `isIssuedByNetwork` (boolean) indicates the authenticator is a certificate that was issued by the network
+- `isExtendRequested` (boolean) indicated the authenticator will report to the client to extend its certificate on next
+  authentication.
+- `isKeyRollRequested` (boolean) indicates if key rolling is requested with an outstanding `isExtendRequested`
+- `extendRequestedAt` (string, date time) indicates when an outstanding extension was requested, otherwise `null`
+  or not provided
+- `lastAuthResolvedToRoot` (boolean) indicates if the last time the client authenticated, it provided a certificate
+  chain that resolved to the root CA. Only valid if `isIssuedByNetwork` is `true`.
+
+### Re-Enrollment
+
+Re-enrollment can be accomplished via the CLI or the Edge Management API. Re-enrollment removes the current authenticator,
+stopping the client from authenticating. All of the underlying configuration for the related identity is preserved. 
+The newly created enrollment will have a new enrollment JWT that will be consumed during the enrollment process and 
+will result in a new client certificate and chain.
+
+**CLI:**
+
```

Comment:
Does this need to be authenticator specific? We have `ziti edge re-enroll edge-router`, it's not possible to have `ziti edge re-enroll identity`?